python/graphframes/graphframe.py

```diff
@@ -205,6 +207,37 @@ def bfs(self, fromExpr, toExpr, edgeFilter=None, maxPathLength=10):
         jdf = builder.run()
         return DataFrame(jdf, self._sqlContext)
 
+    def aggregateMessages(self, aggCol, msgToSrc=None, msgToDst=None):
+        """
+        Aggregates messages from the neighbours.
+
+        When specifying the messages and aggregation function, the user may reference columns using
+        the static methods in :class:`AggregateMessages`.
+
+        See Scala documentation for more details.
+
+        :param aggCol: `pyspark.sql.column.Column` for the requested aggregation output
```

Comment:
added support for String SQL expression in `AggregateMessages.agg()` in Scala and Python